src/components/Layout/Navbar/NavbarMobile.tsx

```diff
@@ -1,79 +1,59 @@
-import { RootState } from '@/src/store';
-import { Box, Button, ListItem, Text, UnorderedList } from '@chakra-ui/react';
-import Image from 'next/image';
-import { useSelector } from 'react-redux';
-import UserProfile from './components/UserProfile';
-import GithubLogin from './components/GithubLogin';
-import { NAV_LINKS } from './NavbarConstant';
-import Link from 'next/link';
-import { Dispatch, SetStateAction, useState } from 'react';
+import { Box, Button, ListItem, Text, UnorderedList } from "@chakra-ui/react";
+import Image from "next/image";
+import { NAV_LINKS } from "./NavbarConstant";
+import Link from "next/link";
+import { Dispatch, SetStateAction, useState } from "react";
+import { UserProfileWithGitHubLogin } from "./components/UserProfileWithGitHubLogin";
 
-export default function NavbarMobile({ isDropdownVisible, setIsDropdownVisible } : {
+import styles from "./navbar.module.css";
+
+export default function NavbarMobile({
+  isLoggedIn,
+  firstName,
+  imageURL,
+  isDropdownVisible,
+  setIsDropdownVisible,
+}: {
+  isLoggedIn: boolean;
   isDropdownVisible: boolean;
+  firstName: string | null;
+  imageURL: string | null;
   setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
```

Comment:
since this types are also used in NavbarDesktop component we can reuse this types